migration_helpers/migration_helpers.py

```diff
@@ -0,0 +1,410 @@
+                spoken_form, value = line.split(",")
+            else:
+                value, spoken_form = line.split(",")
+
+            value = value.strip()
+            if config.get("custom_value_converter"):
+                value = config["custom_value_converter"](value)
+
+        else:
+            spoken_form = value = line.strip()
+
+        spoken_form = spoken_form.strip()
+        output.append(f"{spoken_form}: {value}")
+
+    return "\n".join(output)
+
+
+def write_to_file(filename, text):
+    """
+    Write a text string into a specified file.
+
+    Args:
+    - filename (str): The path of the file where the text should be written.
+    - text (str): The text to be written to the file.
+
+    Returns:
+    - None
+    """
+    with open(filename, "w") as file:
+        file.write(text)
+
+
+def get_new_absolute_path_by_prefixing_dot(absolute_path):
+    """
+    Return the new absolute path for a file by prefixing its base name with a "."
+
+    Args:
+    - absolute_path (str): The absolute path to the file.
+
+    Returns:
+    - str: The new absolute path after prefixing the base name with ".".
+    """
+    dir_name = os.path.dirname(absolute_path)
+    base_filename = os.path.basename(absolute_path)
+
+    # Prefix the base filename with "."
+    new_filename = "." + base_filename
+
+    # Construct the new absolute path
+    new_absolute_path = os.path.join(dir_name, new_filename)
+
+    return new_absolute_path
+
+
+def strip_base_directory(base_dir, path):
+    """
+    Strip the base directory from the path if it exists.
+
+    Parameters:
+    - base_dir (str): The base directory to be stripped.
+    - path (str): The path from which the base directory should be stripped.
+
+    Returns:
+    - str: Path after stripping the base directory.
+    """
+
+    # Ensure both the base directory and path are normalized
+    base_dir = os.path.normpath(base_dir)
+    path = os.path.normpath(path)
+
+    # Check if the path starts with the base directory
+    if path.startswith(base_dir):
+        # Subtract base_dir length from path
+        return path[len(base_dir) :].lstrip(os.sep)  # Remove any leading separators
+    else:
+        return path
+
+
+def convert_files():
+    global known_csv_files
+    known_csv_files = {
+        normalize_path(key): value for key, value in known_csv_files.items()
+    }
+
+    directory_to_search = parent_directory_of_script()
+    print(f"Base directory: {directory_to_search}")
+    csv_relative_files_list = find_csv_files(directory_to_search)
+
+    for csv_relative_file in csv_relative_files_list:
+        csv_file = os.path.join(directory_to_search, csv_relative_file)
+        disabled_csv_file = get_new_absolute_path_by_prefixing_dot(csv_file)
+        if csv_relative_file not in known_csv_files.keys():
+            print(f"Skipping non default csv file {csv_relative_file}")
+            continue
+        config = known_csv_files[csv_relative_file]
+        if not config:
```

Comment:
I don't think this will ever fire, because it would require an entry that is mapped to `False` / `None`, which never occurs. For all the commented out entries in our config, shouldn't we instead map them to `False` / `None` so that they hit this `if` statement?

Reply:
I've opted to remove the unsupported configs and simply leave a comment as to why.